### PR TITLE
feat: Refine the layout of sidebar

### PR DESCRIPTION
**Pull request by Keith-CY**

### CODE REVIEW

File: packages/neuron-ui/src/styles/layout.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
 :root{
-  --sidebar-width: 150px;
+  --sidebar-width: 158px;
```
Comment (ashchan) [inline packages/neuron-ui/src/styles/layout.scss]: What's the sidebar width from the design? A bit wider?

File: packages/neuron-ui/src/locales/en.json [hunk 2]
```diff
@@ -11,7 +11,9 @@
       "history": "History",
       "addresses": "Addresses",
       "settings": "Settings",
-      "nervos-dao": "Nervos DAO"
+      "nervos-dao": "Nervos DAO",
+      "fail-to-fetch-tip-block-number": "Connection error",
+      "sync-not-start": "Synchronization not started yet"
```
Comment (ashchan) [inline packages/neuron-ui/src/locales/en.json]: Will this fit as one line within the sidebar?

File: packages/neuron-ui/src/styles/layout.scss (hunk 1 above)
Comment (Keith-CY) [reply packages/neuron-ui/src/styles/layout.scss]: > What's the sidebar width from the design? A bit wider?

It's from the last design in zeplin.

File: packages/neuron-ui/src/locales/en.json (hunk 2 above)
Comment (Keith-CY) [reply packages/neuron-ui/src/locales/en.json]: > Will this fit as one line within the sidebar?

This message is still wrapped in the sidebar

Comment (ashchan) [reply packages/neuron-ui/src/locales/en.json]: I see. Let's make it shorter.

Comment (ashchan) [inline packages/neuron-ui/src/locales/en.json]:
```suggestion
      "sync-not-start": "Sync not started yet"
```

**Pull request merged**